### PR TITLE
Port default homepage to Protocol (Fixes #8345)

### LINKED ISSUE #8345: Port default homepage to Protocol
## Description

Currently, the default homepage seen by locales that are not in EN, FR, or DE see a template that is using Pebbles (an outdated design system). The page could use an overall redesign, but this issue is to address that at a minimum we should port this template over to Protocol.

Screenshot_2019-12-20  Интернет для людей, а не для прибыли

#### Tasks
- [ ] Template updated.
- [ ] Old assets removed.

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Updates the default homepage to use Protocol.

## Issue / Bugzilla link
#8345

### CODE REVIEW

**Comment by amychurchwell** [comment]
Adding blocked label because this pull requests continues the newsletter Protocol work in #7440, which needs review.

**Comment by amychurchwell** [comment]
Filed issue to provide conditional hero CTA content for users that are already using the Firefox browser. (Is showing "download firefox" to everyone.)
#8359

**Comment by hoosteeno** [comment]
> Filed issue to provide conditional hero CTA content for users that are already using the Firefox browser. (Is showing "download firefox" to everyone.)
> #8359

@amychurchwell, why does that issue block this one? This isn't a regression, right -- the current page shows that message, and the new Protocol page also does -- so entangling them seems unnecessary. Could we land this without a copy/translation cycle?

**Comment by amychurchwell** [comment]
@hoosteeno Copy is not a blocker for this issue- the blocker is from the newsletter work in #7440, which I would like to implement in this PR once it's merged. It is close to being finalized, tying up loose ends.

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 1]
```diff
@@ -38,213 +33,179 @@
...
-
-        <p>
-        {% if l10n_has_tag('home-update-20171116') %}
-          {{ _('With 2x the speed, built-in privacy protection and Mozilla behind it, the new Firefox is the better way to browse.') }}
-        {% else %}
-          {{ _('With 2x the speed, built-in privacy protection and Mozilla behind it, the new Firefox is fast for good.') }}
-        {% endif %}
-        </p>
-
-        {{ download_firefox(dom_id='download-intro', download_location='primary cta') }}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/home.html]: It looks like there's a functional test that is relying on `dom_id='download-intro'` as a locator:

https://github.com/mozilla/bedrock/blob/master/tests/pages/home.py#L13

File: media/css/mozorg/home/home.scss [hunk 2]
```diff
@@ -2,780 +2,94 @@
...
-    }
+.c-column {
+    box-sizing: border-box;
+    display: block;
+    padding: $spacing-xl 0;
 
-    #impact,
-    #innovate {
+    @media #{$mq-lg} {
         float: left;
```
Comment (alexgibson) [inline media/css/mozorg/home/home.scss]: This section needs a little work for RTL languages:

image

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 3]
```diff
@@ -38,213 +33,179 @@
...
+      <ul>
+        <li>
+          {% call picto_card(
+          custom_desc=True,
+          class='extensions') %}
+          <a class="mzp-c-cta-link" rel="external" href="https://addons.mozilla.org/{{ referrals }}">
+            <h2 class="mzp-c-card-picto-title">{{ _('Extensions') }}</h2>
+          </a>
+          <p class="mzp-c-card-picto-desc">{{ _('Personalize Firefox with your favorite extras like password managers,
+            ad blockers and more.') }}</p>
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/mozorg/templates/mozorg/home/home.html]: Yes, agreed. This was an accident from formatting 😅

**Pull request merged**